tests/test_experiment_rsmtool.py

```diff
@@ -120,6 +120,122 @@ def test_run_experiment_lr_subset_feature_file_and_feature_file():
     do_run_experiment(source, experiment_id, config_file)
 
 
+def test_run_experiment_lars():
+
+    # basic experiment with Lars model
+
+    source = 'lars'
+    experiment_id = 'Lars'
+    config_file = join(test_dir,
+                       'data',
+                       'experiments',
+                       source,
+                       '{}.json'.format(experiment_id))
+    do_run_experiment(source, experiment_id, config_file)
+
+    output_dir = join('test_outputs', source, 'output')
+    expected_output_dir = join(test_dir, 'data', 'experiments', source, 'output')
+    html_report = join('test_outputs', source, 'report', '{}_report.html'.format(experiment_id))
+
+    csv_files = glob(join(output_dir, '*.csv'))
+    for csv_file in csv_files:
+        csv_filename = basename(csv_file)
+        expected_csv_file = join(expected_output_dir, csv_filename)
+
+        if exists(expected_csv_file):
+            yield check_file_output, csv_file, expected_csv_file
+
+    yield check_generated_output, csv_files, experiment_id, 'skll'
+    yield check_report, html_report
+
+
+def test_run_experiment_lars_custom_objective():
+
+    # experiment with lars model and a custom tuning objective
+
+    source = 'lars-custom-objective'
+    experiment_id = 'Lars_custom_objective'
+    config_file = join(test_dir,
+                       'data',
+                       'experiments',
+                       source,
+                       '{}.json'.format(experiment_id))
+    do_run_experiment(source, experiment_id, config_file)
+
+    output_dir = join('test_outputs', source, 'output')
+    expected_output_dir = join(test_dir, 'data', 'experiments', source, 'output')
+    html_report = join('test_outputs', source, 'report', '{}_report.html'.format(experiment_id))
+
+    csv_files = glob(join(output_dir, '*.csv'))
+    for csv_file in csv_files:
+        csv_filename = basename(csv_file)
+        expected_csv_file = join(expected_output_dir, csv_filename)
+
+        if exists(expected_csv_file):
+            yield check_file_output, csv_file, expected_csv_file
+
+    yield check_generated_output, csv_files, experiment_id, 'skll'
+    yield check_report, html_report
+
+
+def test_run_experiment_dummyregressor():
```

Comment:
We don't add predictions/model parameters to tests for most SKLL learners because of their non-deterministic nature. But this is not really concern for dummy regressor so we might as well add them.

Reply:
Good point. I will add those.